software/database/src/main/resources/org/apache/brooklyn/entity/database/mssql/installmssql.ps1

```diff
@@ -49,5 +54,9 @@ $exitCode = Invoke-Command -ComputerName $env:COMPUTERNAME -Credential $mycreds
     $process = Start-Process ( $driveLetter + "setup.exe") -ArgumentList "/ConfigurationFile=C:\ConfigurationFile.ini" -RedirectStandardOutput "C:\sqlout.txt" -RedirectStandardError "C:\sqlerr.txt" -Wait -PassThru -NoNewWindow
     $process.ExitCode
 } -Authentication CredSSP -ArgumentList $driveLetter
-
-exit $exitCode
+} Catch {
+ Write-Error $_.Exception
+ Write-Host 'Exception logged'
+ exit 1
+}
+exit $exitCode
```

Comment:
@neykov good point.
It works.
```powershell
Write-Host hi
Try {
$b = 3434
} Catch {
 Write-Host 'test'
}
Write-Host $b  # prints 3434
```
I think that at least from closure prospective PowerShell should be user friendly :)